src/elements/content-preview/ContentPreview.js

```diff
@@ -362,11 +362,16 @@ class ContentPreview extends React.PureComponent<Props, State> {
      * @return {void}
      */
     componentDidUpdate(prevProps: Props, prevState: State): void {
-        const { token } = this.props;
+        const { token, previewExperiences } = this.props;
```

Comment:
Should this also be `experiences`? Should the Props type be updated?

Reply:
It was added to props and the type in this PR https://github.com/box/box-ui-elements/pull/2542
We decided that it made sense to name the prop experiences in Content Preview, and previewExperiences in EUA & BUIE. What do you? think https://github.com/box/box-ui-elements/pull/2542#discussion_r617844541

Reply:
ah ok that sounds good